src/helpers/errors.js

```diff
@@ -6,13 +6,10 @@ export function handleErrors({ attributesErrors, errors }, setError) {
   }
```

Comment:
Shouldn't we capitalize this general error too? Cause for what i've seen in the demo is not, but we are doing it for each field